examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairingCommand.java

```diff
@@ -49,7 +49,7 @@ public abstract class PairingCommand extends MatterCommand
   private final StringBuffer mPassword = new StringBuffer();
   private final StringBuffer mOnboardingPayload = new StringBuffer();
   private final StringBuffer mDiscoveryFilterInstanceName = new StringBuffer();
-  private static Logger logger = Logger.getLogger(PairingCommand.class.getName());
+  protected static Logger logger = Logger.getLogger(PairingCommand.class.getName());

```

Comment:
Would it make sense to keep this private, and instead have the subclasses define their own loggers? That way they will have unique tags to help debugging?

Reply:
sounds good, updated, thanks